standalone/ansible/group_vars/servers.yml

```diff
@@ -0,0 +1,33 @@
+ansistrano_deploy_to: "/home/deploy/apps/{{ app_name }}"
+ansistrano_shared_paths:
+  - .bundle
+  - log
+  - tmp/pids
+  - tmp/cache
+  - tmp/sockets
+  - public/system
+  - vendor/bundle
+ansistrano_shared_files:
+  - .env.production
+ansistrano_deploy_via: "git"
+ansistrano_allow_anonymous_stats: no
+ansistrano_git_repo: "https://github.com/simpledotorg/simple-server.git"
+ansistrano_after_symlink_shared_tasks_file: "roles/deploy/hooks/after_symlink.yml"
+postgres:
+  database_name: "{{ deploy_env }}-{{ app_name }}-db"
+
+server:
+  config:
+    features:
+      ENABLE_SMS_NOTIFICATION_FOR_OTP: true
+      ENABLE_PURGE_ENDPOINT_FOR_QA: false
+      ENABLE_PATIENT_FOLLOWUPS: true
+      ENABLE_PHONE_NUMBER_MASKING: false
+      ENABLE_UPDATED_ANALYTICS_VIEWS: true
+      ENABLE_DASHBOARD_VIEW_BY_DISTRICT: true
+      ENABLE_EXOTEL_WHITELIST_API: true
+      ENABLE_CACHED_QUERIES_FOR_DASHBOARD: true
+      ENABLE_DASHBOARD_SNAPSHOT: true
+      ENABLE_GENERATE_ENCOUNTER_ID_ENDPOINT: false
+      ENABLE_SYNC_ENCOUNTERS: false
+      ENABLE_INVITE_ADMINS: true
```

Comment:
Instead of having feature flags in `group_vars/` and secrets in `hosts/`, it might be a good idea to consolidate all these in the role's vars itself. This lets us tie it to `deploy_env` so that we can have separate config per environment (/roles/deploy/vars/staging). Having the secrets under `hosts/` is also not great, since it gets tied to the hosts themselves. 

Keeping the secrets and feature flags in separate files is still a good idea, since changes to secrets are not very often and are also harder to track. Putting feature flags in the secrets file only makes it harder.